src/types/database.ts

```diff
@@ -3,3 +3,11 @@
 export interface PublicView<T> {
   getPublicView(): T
 }
+
+// The returned object from updateMany
+// Refer here: https://mongoosejs.com/docs/api/model.html#model_Model.updateMany
+export interface UpdateManyMeta {
```

Comment:
should use mongoose's own type instead: `mongoose.UpdateWriteOpResult`

Reply:
fixed!